lib/widgets/reusable_widgets/settings_node.dart

```diff
@@ -131,28 +130,49 @@ class _SettingsNodeState extends State<SettingsNode> {
             'The Embedded Node validates all network transactions\n'
             'It may take several hours to fully sync with the network',
             MaterialCommunityIcons.clock,
+            iconColor: Colors.amber,
           ),
         ),
-        const SizedBox(
-          width: 5.0,
-        ),
         Visibility(
-          visible: !kDefaultNodes.contains(widget.node),
-          child: MaterialIconButton(
-            iconData: Icons.edit,
-            onPressed: () {
-              setState(() {
-                _editable = true;
-              });
-            },
-            materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
+          visible: kDefaultCommunityNodes.contains(widget.node),
+          child: const StandardTooltipIcon(
+            'Hardcoded Community Node',
```

Comment:
"Hardcoded" might be a strange word for non-software developers. Could this just say "Community Node"?